avocado/utils/download.py

```diff
@@ -44,12 +44,18 @@ def url_open(url, data=None, timeout=5):
     old_timeout = socket.getdefaulttimeout()
     socket.setdefaulttimeout(timeout)
     try:
+        if proxies and isinstance(proxies, dict):
```

Comment:
No need to test the type. You specify you need a dict and let urllib2 deal with what comes. (it will raise an `AssertionError: proxies must be a mapping`).